packages/expo-sensors/src/Pedometer.ts

```diff
@@ -64,3 +65,28 @@ export async function getStepCountAsync(start: Date, end: Date): Promise<Pedomet
 export async function isAvailableAsync(): Promise<boolean> {
   return await ExponentPedometer.isAvailableAsync();
 }
+
+// @docsMissing
+export async function getPermissionsAsync(): Promise<PermissionResponse> {
+  if (!ExponentPedometer.getPermissionsAsync) {
+    return defaultPermissionsResponse;
+  } else {
+    return await ExponentPedometer.getPermissionsAsync();
+  }
+}
+
+// @docsMissing
+export async function requestPermissionsAsync(): Promise<PermissionResponse> {
+  if (!ExponentPedometer.requestPermissionsAsync) {
+    return defaultPermissionsResponse;
+  } else {
+    return await ExponentPedometer.requestPermissionsAsync();
+  }
+}
+
+const defaultPermissionsResponse: PermissionResponse = {
+  granted: true,
+  expires: 'never',
+  canAskAgain: true,
+  status: PermissionStatus.GRANTED,
+};
```

Comment:
@EvanBacon You will need to run `et gdad` command to update the generated API JSON file, otherwise newly added methods and their types won't appear in the docs. 

Also you probably should copy the generated JSON from `unversioned` directory to the `v41.0.0` so the changes will be backported to the latest release docs.